tests/constructorTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the biblys/isbn package.
+ *
+ * (c) Clément Bourgoin
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use Biblys\Isbn\Isbn as Isbn;
+use PHPUnit\Framework\TestCase;
+
+class testConstructor extends TestCase
+{
+    /**
+     * Should not raise an error
+     */
+    public function testUnknownPrefix()
```

Comment:
Add "invalid Isbn should not throw" assert message and remove comment
+ regression link to GitHub issue